README.md

```diff
@@ -16,6 +16,109 @@ The Wavefront Java Tracing Agent provides application observability without havi
 The Wavefront Java Tracing Agent includes:
 * The [Java OpenTracing SpecialAgent](https://github.com/opentracing-contrib/java-specialagent), which automatically instruments Java applications with traces.
 * The Wavefront OpenTracing Bundle that is implemented in this repository, which sends the tracing data to Wavefront for observability.
```

Comment:
How about: 
The Wavefront OpenTracing Bundle (implemented in this repository), which sends the tracing data to Wavefront.